src/content/post/podfoods-relevance-blog.mdx

```diff
@@ -17,35 +17,34 @@ import QueryDiscussion from '~/components/common/QueryDiscussion.astro';
 
-Keyword search is enough for the vast majority of queries when someone knows what they're looking for, but less so when you're exploring. Pod Foods is really interesting because they are likely to have a significant number of users exploring rather than checking if a specific item is present. The whole point of the company is **to help retailers find new independent brands and help those brands be found**. So if discovery-focused search is not supported well, it's a big deal.
+Keyword search is ideal when you exactly what you're looking for, but the wrong fit for exploring. Pod Foods seemed like an ideal user-experience to us, because their explicit business goal is **to help retailers find new independent brands and help those brands be found**.
 
-Pod Foods is not our customer, but at Trieve we've been wanting an opportunity to showcase our search infrastructure in an e-commerce context. Trieve's vector based search for semantic, fulltext, and hybrid modes is uniquely well-suited to Pod Foods' dataset. So we've built out a demo within a limited mockup of their website.
+Pod Foods is not our customer, but we've built out a demo within a limited mockup of their website to showcase and test our product. 
 
 - **Live demo:** [podfoods.trieve.ai](https://podfoods.trieve.ai)
 - **GitHub repo:** [devflowinc/podfoods-discovery](https://github.com/devflowinc/podfoods-discovery)
 
 ## Relevance Comparisons
 
-I've compiled several relevance comparisons between the existing search in Pod Foods and Trieve's **fulltext** search. Our fulltext search uses **[SPLADE](https://github.com/naver/splade), a sparse neural search retrieval model**.
+Here we show several relevance comparisons between the existing search in Pod Foods and Trieve's AI-enhanced **fulltext** search.
 
 <TextBox>
-The demo also shows two other search types: **semantic**, a dense vector search using cosine distance vectors (here using [Jina english embeddings](https://jina.ai/news/jina-ai-launches-worlds-first-open-source-8k-text-embedding-rivaling-openai)), and **hybrid**, which takes in results from both fulltext and hybrid and re-ranks with a cross-encoder model (here the [BAAI/bge-reranker-large](https://huggingface.co/BAAI/bge-reranker-large)).
+  Our fulltext search uses **[SPLADE](https://github.com/naver/splade), a sparse neural search retrieval model**. There are also two other search types available: **semantic**, a dense vector search using cosine distance powered by [Jina english embeddings](https://jina.ai/news/jina-ai-launches-worlds-first-open-source-8k-text-embedding-rivaling-openai), and a **hybrid**, which takes in results from both fulltext and semantic then merges and ranks them with the [BAAI/bge-reranker-large](https://huggingface.co/BAAI/bge-reranker-large) cross-encoder model.
```

Comment:
Do we want to say: "There are also two other search types available in the demo:"
Add: 'in the demo'